styles.scss

```diff
@@ -0,0 +1,201 @@
+        position: absolute;
+        top: 0;
+        right: 0;
+        background-color: rgba(10, 44, 68, 0.5);
+        color: white;
+        width: 230px;
+        height: 25px;
+        cursor: pointer;
+
+        .TopMenus-span {
+            border-right: 1px solid white;
+            padding: 0 2% 0 2%;
+            font-size: 55%;
+
+            .TopMenusLink {
+                text-decoration: none;
+                color: white;
+            }
+        }
+
+        .TopMenus-span1 {
+            padding: 0 0 0 2%;
+            font-size: 55%;
+
+            .TopMenusLink {
+                text-decoration: none;
+                color: white;
+            }
+        }
+    }
+}
+
+.GridNavSection {
+    display: grid;
+    grid-template-columns: 25% 75%;
+    background-color: rgba(1, 15, 29, 0.8);
+    border: 1px solid rgba(23, 45, 69);
+    border-style: ridge;
+    outline-color: 0px;
+    outline: 1px solid rgba(178, 189, 181, 0.2);
+    outline-offset: -5px;
+
+    .logosection {
+        position: relative;
+
+        .logolink {
+            .logo {
+                cursor: pointer;
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                -ms-transform: translate(-50%, -50%);
+                transform: translate(-50%, -50%);
+                width: 180px;
+            }
+        }
+    }
+
+    .navoptionsGrid {
+        display: flex;
+        flex-wrap: nowrap;
+    }
+
+    .navoptionsGrid>.navoptionsdiv {
+        text-align: center;
+        line-height: 50px;
+        font-size: 30px;
+
+        .nav-a,.nav-aLast {
+            color: rgba(170, 202, 225, 255);
+            text-decoration: none;
+
+            .delimeter,.delimeterfirst {
+                cursor: pointer;
+                font-size: 15px;
+                font-weight: normal;
+                padding: 3px 15px;
+                background: transparent url(./Images/delimeter.png) no-repeat right;
+            }
+
+            .delimeterfirst {
+                background-color: rgb(72, 104, 129, 0.7);
+                border-radius: 5px;
+                color: white;
+            }
+        }
+
+        .nav-aLast {
+            cursor: pointer;
+            font-size: 15px;
+            font-weight: normal;
+            padding: 3px 0 3px 15px;
+        }
+
+    }
+
+}
+
+// TransparentContent
+.Grid-TransperantContent {
+    display: grid;
+    grid-template-columns: auto;
+    background: linear-gradient(transparent, rgba(1, 15, 29, 0.5), transparent, rgba(1, 15, 29, 0.5));
+
+    .TransparentP {
+        font-family: "Roboto Mono", monospace;
+        color: white;
+        font-size: 30px;
+        padding: 0 10px 0 10px;
+        letter-spacing: 5px;
+    }
+}
+
+.MainContent {
+    display: grid;
+    grid-template-columns: 33.33% 33.33% 33.33%;
+    background-color: rgba(1, 15, 29, 0.8);
+    border: 1px solid rgba(23, 45, 69);
+    border-style: ridge;
+    outline-color: 0px;
```

Comment:
color in pixels?

Reply:
removed it as it was not applicable and color can't be in pixels.